schema/notemons.py

```diff
@@ -0,0 +1,17 @@
+QSChapter = {
+    "section_id" : "s",
+    "section_name" : "t",
+    "short_1" : "s",
+    "short_2" : "s"
+}
+
+QSMons = {
+    "enemy_script": "t",
+    "byte_1": "b",
+    "section_id": "s",
+    "mons_id": "s",
```

Comment:
Not sure if this is mons_id or just a sort order

Reply:
I'm not sure either. It's unique for each entry and in ascending order, with one exception (see below)

Reply:
One simple test is to change some of them and see if it's reflected ingame or not

Reply:
I managed to try it now. The field may be unused. Duplicating and swapping values doesn't have any influence on the order that entries appear in the notebook.

Reply:
hmm, there might be some quirks with how the game read data (in cs1, master quartz shuffle acts weird if you continue from a previous save after doing the shuffle when compare to starting a new game). Not sure if it's the case for notebook as well.

Reply:
It's definitely possible. Since the purpose is unknown and would require extensive testing, I changed the heading to a generic "short_1" for now.

Reply:
I'll do some testing when I get home today

Reply:
did some testing with a new file, the short doesn't appear to do anything. So it's good for a merge